python/sglang/srt/models/phi4mm.py

```diff
@@ -466,6 +501,9 @@ def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
             (".self_attn.qkv_proj", ".self_attn.v_proj", "v"),
         ]
         prefix_mapping = {
+            "model.embed_tokens_extend.audio_embed.audio_projection.vision.": "embed_tokens_extend.audio_projection_for_vision.",
```

Comment:
Minor question: have you got a chance to test both vision and audio in the same request? I saw this test case in HF: https://huggingface.co/microsoft/Phi-4-multimodal-instruct/blob/main/sample_inference_phi4mm.py#L214